x-pack/plugins/enterprise_search/public/applications/workplace_search/index.tsx

```diff
@@ -96,13 +96,7 @@ export const WorkplaceSearchConfigured: React.FC<InitialAppData> = (props) => {
         <SourceAdded />
       </Route>
       <Route exact path="/">
-        {errorConnecting ? (
-          <ErrorState />
```

Comment:
This error state isn't going away - I'll handle not found & error states in the final commit/pass that cleans up all the layouts and so forth. Basically it's going to get handled in 1 place at the top-level going forward [like how App Search manages it](https://github.com/elastic/kibana/blob/66c9d801d5cdcd48f0d5ee0472a337763547fed4/x-pack/plugins/enterprise_search/public/applications/app_search/index.tsx#L57-L62).